src/phoenix/trace/fixtures.py

```diff
@@ -146,6 +214,50 @@ def load_example_traces(use_case: str) -> TraceDataset:
     return TraceDataset(json_lines_to_df(download_traces_fixture(fixture)))
 
 
+def get_dataset_fixtures(
+    use_case: str,
+) -> Iterable[DatasetFixture]:
+    return (fixture.load() for fixture in get_trace_fixture_by_name(use_case).dataset_fixtures)
+
+
+def send_dataset_fixtures(
+    endpoint: str,
+    fixtures: Iterable[DatasetFixture],
+) -> None:
+    sleep(5)  # wait for server to start
+    client = Client(endpoint=endpoint)
+    for i, fixture in enumerate(fixtures):
+        try:
+            if i % 2:
+                client.upload_dataset_table(
+                    fixture.dataframe,
+                    input_keys=fixture.input_keys,
+                    output_keys=fixture.output_keys,
+                    metadata_keys=fixture.metadata_keys,
+                    name=fixture.name,
+                    description=fixture.description,
+                )
+            else:
+                with NamedTemporaryFile() as tf:
+                    with open(tf.name, "w") as f:
+                        shutil.copyfileobj(fixture.csv, f)
+                        f.flush()
+                    client.upload_dataset_table(
+                        tf.name,
+                        input_keys=fixture.input_keys,
+                        output_keys=fixture.output_keys,
+                        metadata_keys=fixture.metadata_keys,
+                        name=fixture.name,
+                        description=fixture.description,
+                    )
+        except HTTPError as e:
+            print(e.response.content.decode())
+            pass
```

Comment:
Seems like we should raise here?

Reply:
This is for the uniqueness constraint, so I wanted the loop to continue, and show that all fixtures exist in the db.